components/searchable_channel_list.jsx

```diff
@@ -68,9 +69,10 @@ export default class SearchableChannelList extends React.PureComponent {
 
         if (shouldShowArchivedChannels) {
             archiveIcon = (
-                <div className='more-modal__icon-container'>
-                    <ArchiveIcon className='icon icon__archive'/>
-                </div>
+                <ArchiveOutlineIcon
+                    size={20}
+                    color={'rgb(var(--center-channel-color-rgb))'}
```

Comment:
same `currentColor`

Reply:
Changed the variable usage here to `currentColor` instead for simplicity.